src/panel/PoiPanel.jsx

```diff
@@ -0,0 +1,345 @@
+          <i className="icon-x" />
+        </div>
+      </div>
+      <div className={classnames('poi_panel', { 'poi_panel--card': this.card })}>
+        <div className="poi_panel__content__card">
+          { this.fromCategory &&
+            <div className="poi_panel__close"
+              onClick={ this.fromCategory ? this.backToList : this.closeAction }>
+              <i className="icon-x" />
+            </div>
+          }
+          <div className="poi_panel__description_container">
+            <div>
+              <PoiHeader poi={this.props.poi} />
+              <OpeningHour poi={this.props.poi} />
+            </div>
+            <PoiTitleImage poi={this.props.poi} iconOnly={true} />
+          </div>
+          <div className="poi_panel__content__card__action__container">
+            { this.isDirectionActive &&
+              <button
+                className="poi_panel__content__card__action poi_panel__content__card__action__direction"
+                onClick={this.openDirection}
+              >
+                <span className="icon-corner-up-right" />
+                { _('DIRECTIONS', 'poi panel') }
+              </button>
+            }
+            <button className="poi_panel__content__card__action" onClick={this.showDetail}>
+              <span className="icon-chevrons-right" />
+              { _('SEE MORE', 'poi panel') }
+            </button>
+          </div>
+        </div>
+        <div className="poi_panel__content">
+          <div className="poi_panel__container">
+            <div className="poi_panel__description_container" onClick={this.center}>
+               <PoiHeader poi={this.props.poi} />
+               <PoiTitleImage poi={this.props.poi} iconOnly={false} />
+            </div>
+            <div className="poi_panel__actions">
+              <button className={classnames(
+                  'poi_panel__action',
+                  'poi_panel__actions__icon__store',
+                  {
+                    'icon-icon_star-filled': this.props.poi.stored,
+                    'icon-icon_star': !this.props.poi.stored,
+                  })
+                }
+                onClick={this.toggleStorePoi}
+              >
+                <div>{this.props.poi.stored ? _('SAVED', 'poi') : _('FAVORITES', 'poi')}</div>
+              </button>
+              <button className="poi_panel__action icon-share-2" onClick={this.openShare}>
+                <div>{_('SHARE', 'poi')}</div>
+              </button>
+              {this.isDirectionActive &&
+                <button className="poi_panel__action icon-corner-up-right"
+                  onClick={this.openDirection}>
+                  <div>{_('DIRECTIONS', 'poi')}</div>
+                </button>
+              }
+              {this.props.poi.blocksByType && this.props.poi.blocksByType.phone &&
+                this.renderPhone()}
+            </div>
+            {isAnywherePoi && this.categories &&
```

Comment:
This is exactly the same visual elements as the category list on the service panel, but maybe the factorization can be done in a future PR. In any case, it would be more convenient as a separate component, it would make this one less polluted by category stuff.